views/js/checkout-component-renderer.js

```diff
@@ -504,6 +491,18 @@ jQuery(document).ready(function() {
             }
         }
 
+        function handleOnClick(resolve, reject) {
+            const paymentMethodContainer = this.paymentForm.closest('.adyen-payment');
+            // Show message if button is disabled else if not in progress, hide and resolve
+            if (prestaShopPlaceOrderButton.prop('disabled') && !isPlaceOrderInProgress()) {
+                showRequiredConditionsInfoMessage(paymentMethodContainer);
+                reject(new Error('Terms of service not agreed'));
```

Comment:
is this shown on the frontend or only in the console?

Reply:
Just in the console